util/topsql/reporter/single_target_test.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2022 PingCAP, Inc.
```

Comment:
This pr is to fix the data race. Why add a new test case for other components?

Reply:
Because the test coverage gets down while the data race is being repaired.